openfisca_us_data/datasets/ce/ce.py

```diff
@@ -0,0 +1,280 @@
+import pkgutil
+
+from pandas import DataFrame, concat
+import numpy as np
+import h5py
+import yaml
+
+from openfisca_us_data.utils import US, dataset
+from openfisca_us_data.datasets.ce.raw_ce import RawCE
+
+
+@dataset
+class CE:
+    """OpenFisca-US dataset for the Consumer Expenditure Survey
+
+    The information extracted by RawCE and CE classes is from the "Interview
```

Comment:
@MaxGhenis , somewhere earlier you had asked about documenting use of the FMLI files. Let me know if this is sufficient.

Reply:
thanks, could you also spell out what FMLI abbreviates?

Reply:
Believe it or not, I don't think it's an abbreviation for anything. From [here](https://www.ilo.org/surveyLib/index.php/catalog/350/download/3928), 

> The FMLI file, also referred to as the “Consumer Unit Characteristics and Income” file, contains
> CU characteristics, CU income, and characteristics and earnings of the reference person and of the
> spouse. The file includes weights needed to calculate population estimates and variances.

Let me know if you can find something on the name, or if you have other suggestions about how to describe it.

Reply:
Weird! OK thanks.